src/tailFunc.js

```diff
@@ -0,0 +1,5 @@
+const tail = arr => {
+  return arr.length === 0 ? 'Empty array!' : arr[arr.length-1]
```

Comment:
![image](https://user-images.githubusercontent.com/32430949/86587298-02b88900-bf92-11ea-932b-8440f811102f.png)

you've written `last`

Let's rename it to last and return `undefined` if there are no elements

Also please handle a situation when user provide `null` as an argument (should also return undefined)